mcs/class/corlib/Mono/Codegen.cs

```diff
@@ -0,0 +1,16 @@
+//
+// Mono codegen gateway functions
+//
+//
+
+using System;
+
+namespace Mono {
+	public static class Codegen
+	{
+		[AttributeUsage(AttributeTargets.All)]
+		public class Reflected : Attribute
```

Comment:
Usually attributes are named `SomethingAttribute` and then they can be applied either as `[SomethingAttribute]` or as `[Something]`.